_test/test_mpi/job_dispatcher_common_tests.m

```diff
@@ -161,7 +164,7 @@ function my_delete(varargin)
             
             
             assertTrue(n_failed >= 1);
-            assertEqual(numel(outputs), 3);
+            %assertEqual(numel(outputs), 3);
```

Comment:
dead code

Reply:
probably would not return this check so, ok, removed